### ISSUE TITLE
508 Office request: Update alt text of VA logo in Medallia survey modal

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#54247]
### Point of contact

Martha Wilkes

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The VA 508 Office requests to change the alt-text for the VA logo to be consistent in all usage scenarios. 

<img width="1509" alt="Screen Shot 2023-02-27 at 12 56 06 PM" src="https://user-images.githubusercontent.com/47792423/221668759-7bf89419-85ad-4ab1-9387-e8b9ebaacdb1.png">

Current incorrect alt-text: "Veterans' Affairs Logo and Typeface"

Should be: "VA logo and Seal, U.S. Department of Veterans Affairs"

Similar to the issue described in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12697

### Reproduction steps

1.
2.
3.
...

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by artsymartha68** [comment]
This is a DUP of https://github.com/department-of-veterans-affairs/va.gov-team/issues/57542 
Closing this one.
